src/sql/services/notebook/notebookServiceImpl.ts

```diff
@@ -169,14 +181,25 @@ export class NotebookService extends Disposable implements INotebookService {
 			throw new Error(localize('notebookUriNotDefined', 'No URI was passed when creating a notebook manager'));
 		}
 		let uriString = uri.toString();
-		let manager = this._managers.get(uriString);
-		if (!manager) {
-			manager = await this.doWithProvider(providerId, (provider) => provider.getNotebookManager(uri));
-			if (manager) {
-				this._managers.set(uriString, manager);
+		let manager: INotebookManager[] = this._managers.get(uriString);
```

Comment:
Can we tidy up the names here? Shouldn't this be `managers` since it's an array?